blocks/library/quote/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { registerBlock, query as hpq } from 'api';
+import Editable from 'components/editable';
+
+const { html, query } = hpq;
+
+registerBlock( 'core/quote', {
+	title: wp.i18n.__( 'Quote' ),
+	icon: 'format-quote',
+	category: 'common',
+
+	attributes: {
+		value: ( node ) => query( 'blockquote > p', html() )( node ).join(),
```

Comment:
>The above will retrieve all the paragraphs and join them, so no it won't disallow multiple paragraphs.

But `html` retrieves the `innerHTML` so it would lose the tag and become a single joined string.

>I wanted to have a unique string to pass to the Editable inside the  `edit` instead of having to duplicate the join logic in `edit` and `save`.

I guess I'm more on the side of using `attributes` to get the raw-est data possible even if that's not necessarily the most useful, and assume that `edit` and `save` will form it into whatever's most applicable, perhaps sharing logic with common components if need be.

>Also worth noting that the footer can contain `p` elements

Is this "can" as in: technically possible per spec, or something we want to support?